www_src/components/basic-element/basic-element.jsx

```diff
@@ -40,11 +40,11 @@ var El = React.createClass({
       zIndex: this.props.zIndex,
       touchactive: false
     };
+    this.buttonStyle = initial;
```

Comment:
Agreed: if it's to reset the component, that's a perfect use for getInitialState. That's not what the code did at all, though, it pretended getInitialState was a style definition